docs/shared_parsers_catalog/neutron_plugin.rst

```diff
@@ -0,0 +1,9 @@
+Neutron Plugin Configuration
+============================
+
+file ``/etc/neutron/plugin.ini``
```

Comment:
I think specifying the classes in the `.rst` file is more useful when there are multiple mappers in module.  It doesn't seem to provide much benefit here.